packages/manager/apps/vrack-services/src/components/listing/CreateVrack.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_TYPE,
+  ODS_BUTTON_VARIANT,
+} from '@ovhcloud/ods-components/button';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { ODS_MESSAGE_TYPE } from '@ovhcloud/ods-components/message';
+import { OsdsMessage } from '@ovhcloud/ods-components/message/react';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { useAuthentication } from '@ovh-ux/manager-react-core-application';
+import { CountryCode } from '@ovh-ux/manager-config';
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components/spinner';
+import { OsdsSpinner } from '@ovhcloud/ods-components/spinner/react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import {
+  OrderDescription,
+  getDeliveringOrderList,
+  getDeliveringOrderQueryKey,
+  getVrackListQueryKey,
+  orderVrack,
+  orderVrackQueryKey,
+  useOrderPollingStatus,
+} from '@/api';
+import { DeliveringMessages } from './DeliveringMessages';
+
+export type CreateVrackProps = {
+  closeModal: () => void;
+};
+
+export const CreateVrack: React.FC<CreateVrackProps> = ({ closeModal }) => {
+  const { t } = useTranslation('vrack-services/listing');
+  const { subsidiary } = useAuthentication();
+  const queryClient = useQueryClient();
+  const {
+    data: vrackDeliveringOrders,
```

Comment:
Just a global suggestion about naming variable and readability, as we are in vrack project for me you don't need to prefix/add in all you're function/const the vRack name you can simplify naming like : 

isVrackOrdersError => isOrdersError
areVrackOrdersLoading => areOrdersLoading
vrackDeliveringOrders => deliveringOrders
orderNewVrack => orderNew
getVrackListQueryKey => getListQueryKey
vrackServicesId => servicesId


To avoid confusion and long naming, also sometimes you add vrack at the beginning or at the end

Reply:
Good point but we are in the "vRack Services" project and sometimes we need to query the "vRack" endpoints (order, association, listing etc.). That's why sometimes I need to precise that's it's about "vRack" and sometimes about "vRack Services" (when querying in the same component).

Reply:
Don't really understand difference between this naming (service or not) 
You can just remove the `vRack` label and add `service` if you need it

Reply:
That's 2 different products
One is named `vRack Services` and the other is named `vRack`